src/server/scripts/Spells/spell_rogue.cpp

```diff
@@ -657,10 +709,10 @@ class spell_rog_stealth : public SpellScriptLoader
             }
         };
 
-        AuraScript* GetAuraScript() const override
-        {
-            return new spell_rog_stealth_AuraScript();
-        }
+	AuraScript* GetAuraScript() const
```

Comment:
tab2spaces

Reply:
Sorry dunno what is going on with whitespaces. Its ok in my screen but its broken then... Does someone care in editing that mistakes please?